### ISSUE TITLE
Can't find pytorch3d version 0.7.6 in pytorch3d channel (conda)

### ISSUE THREAD

**Issue by idanshef** [facebookresearch/pytorch3d#1823]
By searching pytorch3d in pytorch3d conda channel, I can find only versions up until 0.7.5
I did find version 0.7.6 with conda-forge, but it ask me to install cpu version of pytorch instead of cuda.
What should I do?

**Comment by bottler** [comment]
Yes, I agree. I'm not sure what happened - 0.7.6 appears not to have conda packages now, and I don't know if it ever did. I expect 0.7.7 to be released very soon and it will have them.
